_posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md

```diff
@@ -0,0 +1,51 @@
+* OpenSearch GenAI Use Cases : Anupam Chaturvedi, Adobe Engineering Manager, demonstrated how Adobe is leveraging OpenSearch for its Acrobat GenAI assistant, showcasing the practical applications of OpenSearch in the generative AI space.
+* Community Growth: Jim Zemlin, Executive Director, ![The Linux Foundation](https://www.linuxfoundation.org/) spoke about welcoming OpenSearch to the Linux Foundation. By opening up the project governance and moving to a neutral home, the OpenSearch team is taking concrete steps to further the project’s founding principle of being for the community and by the community. 
+
+3. Important Sessions and Workshops
+The conference featured several informative sessions and workshops, including:
+
+* Session: ![Airbnb Embedding Platform](https://www.youtube.com/watch?v=kpWQ55hVF90)
+    This session, led by Moutupsi Paul, Xiaotang Wang and Amulya Sharma, focused on how Airbnb is leveraging OpenSearch as a vector database for its embedding platform. The session provided an overview of how Airbnb is using vector embeddings to solve guest, host & marketplace challenges, followed by specific use cases that led Airbnb to adopt OpenSearch as its vector database. The session shared lessons learned and provided insights into strategies for optimizing OpenSearch as a vector database.
+* Session: ![Lucene And Beyond - Core Storage Extension In OpenSearch](https://youtu.be/-_mXJqvPw0o?si=7Lb1Y__uAZ7ZCc66)
+    OpenSearch is tightly bound to the Lucene core APIs that facilitate Encoding, Transactions, Merges, Search. This session, led by Samuel Herman, focused on how the OpenSearch storage encoding can be extended to popular formats (e.g. Parquet, Avro) that are readable by public big data systems such as Apache spark. 
+* Session: ![Bringing cloud native innovation from search platforms at Uber and Slack into OpenSearch](https://youtu.be/iZrFVqvTD9s?si=loButiXx8zm2PHob)
+    This session, led by Karthikeyan Ramasamy and Bryan Burkholder, showcased how their organizations’ unique requirements of low latency, high throughput workloads led them to develop their own search and log analytics platforms, making it easy to operate, cost-effective, and scale to petabytes of data. The session covers how Uber and Slack are collaborating with OpenSearch Project to bring some of their learnings and innovations from their platforms into OpenSearch, with special focus on a modular cloud native architecture with isolation between readers and writers and a performant messaging protocol for communication between services. 
+* Session: ![A deep dive into OpenSearch Serverless](https://www.youtube.com/watch?v=JPo_uF4Fw4M)
+    This session led by Rohit Nair and Milav Shah, dives deep into how OpenSearch serverless works, including architecture details, understanding performance and operational characteristics and future roadmap.
```

Comment:
This session led by Rohit Nair and Milav Shah, dives deep into how Amazon OpenSearch Serverless works, including architecture details, understanding performance and operational characteristics and future roadmap.

Reply:
@prthsh2020 Just need a comma after "session", another after "characteristics", and "the" before "future".